contracts/SL2RD.sol

```diff
@@ -315,15 +315,7 @@ contract SL2RD is
             "SHARE007"
         );
 
-        // Allows the operator to call transfer without revert
-        if (
-            _shareOperatorRegistry.isOperator(msg.sender) ||
-            msg.sender == owner()
-        ) {
-            super._approve(msg.sender, _nextAvailableCommunitySlot);
-        }
-
-        super.transferFrom(owner(), to_, _nextAvailableCommunitySlot);
+        super._transfer(splitOwner, to_, _nextAvailableCommunitySlot);
```

Comment:
How does super._transfer guarantee that an operator address can execute this call? I haven't looked at the `_transfer` implementation but in the prior version we called super._approve before executing transfer from, because the sender of this transaction may not be the contract owner but instead an operator EOA from the network pool.

Reply:
This was written a while back so I am trying to recall why I made this change. I believe it is because the function itself can only be called by an owner or operator, so there would be no reason to ensure that the caller is in the operator registry. The old logic also assumes the owner of the contract is the owner of the slot, but the point of this PR is that you can have multiple initial owners so the owner of the contract may not be the owner of the slot.